Geometry/TrackerNumberingBuilder/interface/GeometricDet.h

```diff
@@ -81,128 +80,118 @@ class GeometricDet {
     PixelPhase2TDRDisk = 237
   } GeometricEnumType;
 
-  /**
-   * Constructors to be used when looping over DDD
-   */
+  // Constructors from Filtered View (called while looping over DD).
   GeometricDet(DDFilteredView* fv, GeometricEnumType dd);
   GeometricDet(cms::DDFilteredView* fv, GeometricEnumType dd);
+  // Constructors from persistent data (from DB)
   GeometricDet(const PGeometricDet::Item& onePGD, GeometricEnumType dd);
 
-  /**
-   * set or add or clear components
-   */
-  void setGeographicalID(DetId id) { _geographicalID = id; }
+  // ACCESS GENERAL INFO
+  const std::string& name() const { return ddname_; }
+  const GeometricEnumType& type() const { return type_; }
+
+  // NAVIGATION related info
+  const nav_type& navType() const { return ddd_; }
+  NavRange navpos() const { return NavRange(&ddd_.front(), ddd_.size()); }
+  const DetId& geographicalId() const { return geographicalID_; }
```

Comment:
@cvuosalo is unified here

Reply:
@cvuosalo Yes basically there were 2 member functions with different names, returning the same member data. I cleaned it on the fly.